router/router.go

```diff
@@ -1540,12 +1590,15 @@ func (rt *HandleT) commitStatusList(responseList *[]jobResponseT) {
 			rt.MultitenantI.CalculateSuccessFailureCounts(workspaceID, rt.destName, true, false)
 			completedJobsList = append(completedJobsList, resp.JobT)
 		case jobsdb.Aborted.State:
-			routerWorkspaceJobStatusCount[workspaceID] += 1
 			sd.Count++
-			rt.MultitenantI.CalculateSuccessFailureCounts(workspaceID, rt.destName, false, true)
 			routerAbortedJobs = append(routerAbortedJobs, resp.JobT)
 			PrepareJobRunIdAbortedEventsMap(resp.JobT.Parameters, jobRunIDAbortedEventsMap)
 			completedJobsList = append(completedJobsList, resp.JobT)
+			routerWorkspaceJobStatusCount[workspaceID] += 1
+			rt.MultitenantI.CalculateSuccessFailureCounts(workspaceID, rt.destName, false, true)
+		case utilTypes.DiffStatus:
+			sd.Count--
+			routerWorkspaceJobStatusCount[workspaceID] += 1
```

Comment:
Saving the failed record for such a job would spur the sources pod to retry such an event wouldn't it?
adding to `jobRunIDAbortedEventsMap` would cause this.
The `completedJobsList` would be required because it's used for `rsources_stats`, but that would involve the above problem as well.
This happens in processor as well btw, shouldn't we be doing the same here...?

Reply:
yeah
we can only add the job to `completedJobsList`.
`jobRunIDAbortedEventsMap` is not relevant for the stats anyway.